pkg/plugin/authorization/http.go

```diff
@@ -0,0 +1,36 @@
+package authorization
+
+import (
+	"errors"
+	"io"
+	"net"
+	"net/http"
+	"time"
+)
+
+var (
+	ErrTimeout = errors.New("timed out")
+)
+
+// doGetRequestWithTimeout sends GET http request on passed URL with 3 sec timeout.
+func doGetRequestWithTimeout(url string) ([]byte, error) {
+	client := http.Client{Timeout: 3 * time.Second}
```

Comment:
I think timeout in this func should be an argument. Something like this:
```
doGetRequestWithTimeout(url string, timeoutDuration int) ([]byte, error)
```

Reply:
Nice adjustment, thanks